src/main/scala/Garage/MongoCRUD.scala

```diff
@@ -0,0 +1,105 @@
+package Garage
+
+import org.mongodb.scala.MongoClient
+import org.bson.{BsonDocument, BsonArray}
+import org.mongodb.scala._
+import org.mongodb.scala.model.Filters.{equal, _}
+import org.mongodb.scala.model.Updates._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.{Failure, Success}
+
+object MongoCRUD extends DBConnection {
+  def openVehicleConnection(): (MongoClient, MongoCollection[Document]) ={
+    val mongoClient = getClient()
+    val db = getDatabase(mongoClient, "garage")
+    (mongoClient, getCollection(db, "vehicles"))
```

Comment:
why not just say `(getClient(), getCollection(getDatabase(mongoClient, "garage"), "vehicles"))`